scripts/tests/test_apple_package.py

```
@@ -0,0 +1,37 @@
+    def test_rejects_coreai_reference_with_multiple_at_signs(self):
+        environment = os.environ.copy()
+        environment.update(
+            {
+                "MESH_APPLE_COREAI_MODEL_REF": (
+                    "owner/repository@mutable@" + "a" * 40
+                ),
+                "MESH_APPLE_COREAI_MODEL_ID": "owner/repository",
+                "MESH_APPLE_COREAI_MODEL_VERSION": "1.0.0",
+            }
+        )
+        result = subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)